responses/41-final-tips.md

```diff
@@ -0,0 +1,12 @@
+
+You are nearly done with the second pipelines course! We have a few final thoughts before you level up your pipelines capability :trophy:
+
+---
+
+First, you may have seen a folder called `_targets` :file_folder: created after running `tar_make()` for the first time. This folder contains the intermediate objects built when running the pipeline, as well as, status metadata about what parts of the pipeline have been built or need to be rebuilt. This kind of information is extremely important to _you_ on _your_ computer, but your collaborators will have unique status information on their own computer. For this reason, make sure to add `_targets/*` to your `.gitignore` file so that it won't be committed.
```

Comment:
```suggestion
First, you may have seen a folder called `_targets` :file_folder: created after running `tar_make()` for the first time. This folder contains the intermediate objects built when running the pipeline, as well as status metadata about the inputs from which each target was most recently built. This kind of information is extremely important to _you_ on _your_ computer, but your collaborators will have unique status information on their own computer. For this reason, make sure to add `_targets/*` to your `.gitignore` file so that it won't be committed.
```

> status metadata about what parts of the pipeline have been built or need to be rebuilt
I think that's not quite true and have revised, but I haven't checked the targets docs or _targets folder recently, so if you're feeling confident, feel free to correct me / revert the text.

Reply:
Hmmm I don't think mine is accurate, but I'm not sure if your change captures it either. I see `meta`, `progress`, and `process` in the `_targets/meta` folder. `meta` looks like this when you open it:
```
name|type|data|command|depend|seed|path|time|size|bytes|format|iteration|parent|children|seconds|warnings|error
site_data|stem|7548fd3496ba58c9|63dc1fe8030252dd|feb4d1733c09582e|1402001415|_targets/objects/site_data|36d26877c46ac716|8cb3d62b9d199995|673437|rds|vector|||24.24|The path argument of write_csv is deprecated as of readr 1.4.0.Please use the file argument instead.90mThis warning|
site_data_clean|stem|0dca29e6e7f7ac1d|532cc5939add5f3f|1a25669c08f2a9e0|1868420800|_targets/objects/site_data_clean|9015cf8c56bc1232|e16b4eb89fb9b3da|656218|rds|vector|||0.02||
site_data_annotated|stem|4aa61e3ae970c1cc|e626a834b6fea8fb|6efd77caf84664d0|2015992021|_targets/objects/site_data_annotated|b65c1a4540a98698|5fe7a02d58ff8301|683687|rds|vector|||0.43||
```

So, perhaps I will change to `status metadata about the most recent build for each target`

Reply:
I like that change 👍